README.md

```
@@ -125,6 +125,31 @@ cargo run -- --export
 cargo run -- --export=./reports --json
 ```
 
+#### CLI JSON/error contract
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)